examples/platform/openiotsdk/app/openiotsdk_platform.cpp

```diff
@@ -277,3 +288,50 @@ int openiotsdk_network_init(bool wait)
 
     return EXIT_SUCCESS;
 }
+
+int openiotsdk_chip_run(void)
+{
+    CHIP_ERROR err;
+
+#ifdef USE_CHIP_DATA_MODEL
+    // Init ZCL Data Model and start server
+    static chip::CommonCaseDeviceServerInitParams initParams;
+    err = initParams.InitializeStaticResourcesBeforeServerInit();
+    if (err != CHIP_NO_ERROR)
+    {
+        ChipLogError(NotSpecified, "Initialize static resources before server init failed: %s", err.AsString());
+        return EXIT_FAILURE;
+    }
+    initParams.operationalServicePort        = CHIP_PORT;
+    initParams.userDirectedCommissioningPort = CHIP_UDC_PORT;
+
+    err = Server::GetInstance().Init(initParams);
```

Comment:
would it make sense to pass the parameters in the function argument ? Why are they static ?

Reply:
No, we use the same initParams in each case.  I see no reason to pass it as an argument. As I understand `CommonCaseDeviceServerInitParams` statically owns and initializes some server sources so it should be only a single instance of it (just in case someone call `openiotsdk_chip_run()` twice). It's a common Matter approach for server initialization so I prefer to stay with it.